prime-router/src/test/kotlin/ReceiverTests.kt

```diff
@@ -12,6 +13,43 @@ import kotlin.test.Test
 internal class ReceiverTests {
     private val translatorConfig = mockkClass(TranslatorConfiguration::class)
 
+    @Test
+    fun `test condition filter on non full-elr pipeline`() {
+        val receiver = Receiver(
+            name = "elr",
+            organizationName = "IGNORE",
+            topic = Topic.COVID_19,
+            customerStatus = CustomerStatus.INACTIVE,
+            conditionFilter = listOf("blah"),
+            translation = translatorConfig,
+            externalName = "Ignore ELR"
+        )
+
+        assertThat(
+            receiver.consistencyErrorMessage(
+                UnitTestUtils.simpleMetadata
+            )
+        ).isEqualTo(
+            "Invalid value for condition filter. " +
+                "Cannot have a condition filter in pipelines other than universal pipeline."
+        )
+    }
+
+    @Test
+    fun `test condition filter on full-elr pipeline`() {
+        val receiver = Receiver(
+            name = "elr",
+            organizationName = "IGNORE",
+            topic = Topic.FULL_ELR,
+            customerStatus = CustomerStatus.INACTIVE,
+            conditionFilter = listOf("blah"),
+            translation = translatorConfig,
+            externalName = "Ignore ELR"
+        )
+
+        assertThat(receiver.consistencyErrorMessage(UnitTestUtils.simpleMetadata)).isEqualTo(null)
```

Comment:
I see both `isEqualTo(null)` and `isNull()` used in the repo, but `isNull()` is probably a little more concise.